### PR TITLE
Feature/df 2283 avam rav importer

### PR DESCRIPTION
**Pull request by birom**

### CODE REVIEW

File: alv-avam-rav-importer/src/main/java/ch/admin/seco/alv/basis/rav/importer/DatasourceConfig.java [hunk 1]
```diff
@@ -31,28 +31,28 @@ public DataSource batchDataSource() {
...
+    @ConfigurationProperties("avam-rav-importer.datasource.basis-batches")
+    public DataSourceProperties basisBatchesDataSourceProperties() {
         return new DataSourceProperties();
     }
 
     @Bean
     @LiquibaseDataSource
     public DataSource lamdaDataSource() {
-        return lamdaDataSourceProperties()
+        return basisBatchesDataSourceProperties()
```
Comment (emina-str) [inline alv-avam-rav-importer/src/main/java/ch/admin/seco/alv/basis/rav/importer/DatasourceConfig.java:42]: Would be good to rename also the lamdaDataSource and lamdaDataSourceInitializer then?

File: alv-basis-batches-database/src/main/resources/liquibase/changelog/20200417120000_add_avam_rav_importer_tables.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```
Comment (emina-str) [inline alv-basis-batches-database/src/main/resources/liquibase/changelog/20200417120000_add_avam_rav_importer_tables.xml:1]: We don't need `AUX_SBALL` view? It had query from `aux_rav_sachb`, `aux_rav_sachb_aab` and `aux_ravadr` tables?

**Comment by emina-str** [comment]
I think we can delete `aux_ravadr.csv`. Loading data from the csv was only a temporary solution.

File: alv-avam-rav-importer/src/main/java/ch/admin/seco/alv/basis/rav/importer/DatasourceConfig.java (hunk 1 above)
Comment (birom) [reply alv-avam-rav-importer/src/main/java/ch/admin/seco/alv/basis/rav/importer/DatasourceConfig.java:42]: Thx, I've fixed it.

File: alv-basis-batches-database/src/main/resources/liquibase/changelog/20200417120000_add_avam_rav_importer_tables.xml (hunk 2 above)
Comment (birom) [reply alv-basis-batches-database/src/main/resources/liquibase/changelog/20200417120000_add_avam_rav_importer_tables.xml:1]: thx, fixed

**Pull request merged**